_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -29763,6 +29758,12 @@
 	},
 /area/ai_monitored/storage/eva)
 "biO" = (
+/obj/structure/cable{
+	d1 = 1;
+	d2 = 2;
+	icon_state = "1-2";
+	tag = ""
```

Comment:
```suggestion
	icon_state = "1-2"
```

Reply:
All of these were there already but yeah, why not remove them.